pkg/config/config.go

```diff
@@ -39,9 +39,9 @@ func override(v *viper.Viper) {
 		"synthetic-monitoring.logs-id":    "GRAFANA_SM_LOGS_ID",
 		"synthetic-monitoring.metrics-id": "GRAFANA_SM_METRICS_ID",
 
-		"mimir.address":   "CORTEX_ADDRESS",
-		"mimir.tenant-id": "CORTEX_TENANT_ID",
```

Comment:
Can we keep retrocompatibility (support two envvars at the same time)? We may have people using these envvars with Mimir or Grafana Cloud. This is a breaking change